find module 'tslib'

```
internal/modules/cjs/loader.js:983
  throw err;
  ^

Error: Cannot find module 'tslib'

```

i dont know what happen, i just install by npm and  use it like example, but i get the error

I'm seeing the same thing when used in an AWS Lambda function.  Not sure if this is a Lambda environment issue or something else.

```
2021-01-15T21:47:55.248Z	undefined	ERROR	Uncaught Exception 	
{
    "errorType": "Runtime.ImportModuleError",
    "errorMessage": "Error: Cannot find module 'tslib'\nRequire stack:\n- /var/task/node_modules/node-2fa/dist/index.js\n- /var/task/index.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js",
    "stack": [
        "Runtime.ImportModuleError: Error: Cannot find module 'tslib'",
        "Require stack:",
        "- /var/task/node_modules/node-2fa/dist/index.js",
        "- /var/task/index.js",
        "- /var/runtime/UserFunction.js",
        "- /var/runtime/index.js",
        "    at _loadUserApp (/var/runtime/UserFunction.js:100:13)",
        "    at Object.module.exports.load (/var/runtime/UserFunction.js:140:17)",
        "    at Object.<anonymous> (/var/runtime/index.js:43:30)",
        "    at Module._compile (internal/modules/cjs/loader.js:1015:30)",
        "    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)",
        "    at Module.load (internal/modules/cjs/loader.js:879:32)",
        "    at Function.Module._load (internal/modules/cjs/loader.js:724:14)",
        "    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)",
        "    at internal/main/run_main_module.js:17:47"
    ]
}
```

--- a/package.json
+++ b/package.json
@@ -37,7 +37,8 @@
   "homepage": "https://github.com/jeremyscalpello/node-2fa#readme",
   "dependencies": {
     "notp": "^2.0.3",
-    "thirty-two": "1.0.2"
+    "thirty-two": "1.0.2",
+    "tslib": "^2.1.0"
   },
   "devDependencies": {
     "@types/jest": "^26.0.20",
@@ -49,7 +50,6 @@
     "jest": "^26.6.3",
     "prettier": "^2.2.1",
     "ts-jest": "^26.4.4",
-    "tslib": "^2.1.0",
     "typescript": "^4.1.3"
   },
   "prettier": {

